docs/reference/runtimes/python/python-reference.md

```diff
@@ -88,12 +88,13 @@ COPY . /opt/nuclio
 CMD [ "processor" ]
 ```
 
-## Supported versions:
+## Supported versions
 * [Python 3.7](#python-runtimes-37-38) - will be fully deprecated soon
 * [Python 3.8](#python-runtimes-37-38) - will be fully deprecated soon
```

Comment:
need to add deprecation of python 3.9 as well, perhaps not this pr but we need to enhance the doc asap.

Reply:
@liranbg this should be done in development